react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -253,16 +421,31 @@ class Thumbnail extends Component<Props, State> {
      * @returns {Object} - The styles for the thumbnail.
      */
     _getStyles(): Object {
-        const { _height, _heightToWidthPercent, _currentLayout } = this.props;
-        let styles;
+        const { _height, _heightToWidthPercent, _currentLayout, _isHidden, _width } = this.props;
+        const userAgent = window.navigator.userAgent;
+        const isOverflowHidden = userAgent.indexOf('QtWebEngine') > -1
```

Comment:
What is this for? QtWebEngine is Chrome these days.

Reply:
We had this in the legacy code. Didn't know that! I'm removing it!